releasenotes/notes/refactor-receiveresourcespans-1a7a1b027982f39c.yaml

```diff
@@ -0,0 +1,10 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - Added a new feature flag to enable a refactored implementation of ReceiveResourceSpans for OTLP - add `enable_receive_resource_spans_v2` in DD_APM_FEATURES.
```

Comment:
```suggestion
  - Added a new feature flag to enable a refactored implementation of ReceiveResourceSpans for OTLP.
  - Added `enable_receive_resource_spans_v2` in DD_APM_FEATURES.
```

Reply:
Based on your description, it sounds like these two features are separate (although related). As such, suggesting to separate them a bit more to clarify. Let me know if this is not accurate.

Reply:
There's actually just one new flag, I rewrote it to try and clarify.